tests/test_convert_lit_checkpoint.py

```diff
@@ -101,3 +102,45 @@ def test_against_original_gpt_neox():
     ours_y = ours_model(x)
     theirs_y = theirs_model(x)["logits"]
     torch.testing.assert_close(ours_y, theirs_y)
+
+
+def test_maybe_unwrap_state_dict(tmp_path):
+    from lit_gpt import Config, GPT
+    from scripts.convert_lit_checkpoint import convert_lit_checkpoint
+    from finetune.full import save_checkpoint
+
+    # fabric is needed for finetune.full::save_checkpoint
+    fabric = L.Fabric(devices=1)
+
+    ckpt_path: Path = tmp_path / "lit_model_finetune.pth"
+    ckpt_name = ckpt_path.name
+
+    model_name = "pythia-1b"
```

Comment:
Let's use the smallest model possible here to make the test fast. For this test, model size is irrelevant.

```suggestion
    model_name = "pythia-70m"
```